### PR TITLE
pgwire: avoid mentioning "tenants" in pre-serve metric descriptions

### PR DESCRIPTION
**Pull request by knz**
Found via #109042.
Epic: CRDB-29380
Release note: None

### CODE REVIEW

File: pkg/sql/pgwire/pre_serve.go [hunk 1]
```diff
@@ -39,37 +39,37 @@ import (
...
 		Name:        "sql.pre_serve.new_conns",
-		Help:        "Counter of the number of SQL connections created prior to routine the connection a specific tenant",
+		Help:        "Counter of the number of SQL connections created prior to routing the connection to the target SQL server",
 		Measurement: "Connections",
 		Unit:        metric.Unit_COUNT,
 	}
 	MetaPreServeBytesIn = metric.Metadata{
 		Name:        "sql.pre_serve.bytesin",
-		Help:        "Number of SQL bytes received prior to routing the connection to a specific tenant",
+		Help:        "Number of SQL bytes received prior to routing the connectionto the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:48]:
```suggestion
		Help:        "Number of SQL bytes received prior to routing the connection to the target SQL server",
```

File: pkg/sql/pgwire/pre_serve.go [hunk 2]
```diff
@@ -39,37 +39,37 @@ import (
 var (
 	MetaPreServeNewConns = metric.Metadata{
 		Name:        "sql.pre_serve.new_conns",
-		Help:        "Counter of the number of SQL connections created prior to routine the connection a specific tenant",
+		Help:        "Counter of the number of SQL connections created prior to routing the connection to the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:42]: For consistency with the others, you could do:

```suggestion
		Help:        "Number of SQL connections created prior to routing the connection to the target SQL server",
```

File: pkg/sql/pgwire/pre_serve.go [hunk 3]
```diff
@@ -39,37 +39,37 @@ import (
...
 		Name:        "sql.pre_serve.bytesin",
-		Help:        "Number of SQL bytes received prior to routing the connection to a specific tenant",
+		Help:        "Number of SQL bytes received prior to routing the connectionto the target SQL server",
 		Measurement: "SQL Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
 	MetaPreServeBytesOut = metric.Metadata{
 		Name:        "sql.pre_serve.bytesout",
-		Help:        "Number of SQL bytes sent prior to routing the connection to a specific tenant",
+		Help:        "Number of SQL bytes sent prior to routing the connectionto the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:54]:
```suggestion
		Help:        "Number of SQL bytes sent prior to routing the connection to the target SQL server",
```

File: pkg/sql/pgwire/pre_serve.go [hunk 4]
```diff
@@ -39,37 +39,37 @@ import (
...
 		Measurement: "SQL Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
 	MetaPreServeConnFailures = metric.Metadata{
 		Name:        "sql.pre_serve.conn.failures",
-		Help:        "Number of SQL connection failures prior to routing the connection to a specific tenant",
+		Help:        "Number of SQL connection failures prior to routing the connectionto the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:60]:
```suggestion
		Help:        "Number of SQL connection failures prior to routing the connection to the target SQL server",
```

File: pkg/sql/pgwire/pre_serve.go [hunk 5]
```diff
@@ -39,37 +39,37 @@ import (
...
 		Measurement: "Connections",
 		Unit:        metric.Unit_COUNT,
 	}
 	MetaPreServeMaxBytes = metric.Metadata{
 		Name:        "sql.pre_serve.mem.max",
-		Help:        "Memory usage for SQL connections prior to routing the connection to a specific tenant",
+		Help:        "Memory usage for SQL connections prior to routing the connectionto the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:66]:
```suggestion
		Help:        "Memory usage for SQL connections prior to routing the connection to the target SQL server",
```

File: pkg/sql/pgwire/pre_serve.go [hunk 6]
```diff
@@ -39,37 +39,37 @@ import (
...
 		Measurement: "Memory",
 		Unit:        metric.Unit_BYTES,
 	}
 	MetaPreServeCurBytes = metric.Metadata{
 		Name:        "sql.pre_serve.mem.cur",
-		Help:        "Current memory usage for SQL connections prior to routing the connection to a specific tenant",
+		Help:        "Current memory usage for SQL connections prior to routing the connectionto the target SQL server",
```
Comment (stevendanna) [inline pkg/sql/pgwire/pre_serve.go:72]:
```suggestion
		Help:        "Current memory usage for SQL connections prior to routing the connection to the target SQL server",
```

**Review by stevendanna — CHANGES_REQUESTED** [review]

**Review by knz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/109076)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stevendanna and @yuzefovich)

___
*[`pkg/sql/pgwire/pre_serve.go` line 42 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299280976:-NcM214TDB1st9MGqjVX:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L42)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

For consistency with the others, you could do:

```suggestion
		Help:        "Number of SQL connections created prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.
___
*[`pkg/sql/pgwire/pre_serve.go` line 48 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299280929:-NcM21JfBoSN7Rphy8mv:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L48)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

```suggestion
		Help:        "Number of SQL bytes received prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.
___
*[`pkg/sql/pgwire/pre_serve.go` line 54 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299280982:-NcM21fsD5Mf-bE4VmZ5:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L54)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

```suggestion
		Help:        "Number of SQL bytes sent prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.
___
*[`pkg/sql/pgwire/pre_serve.go` line 60 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299281006:-NcM21y_0J1JsgOlRiEq:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L60)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

```suggestion
		Help:        "Number of SQL connection failures prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.
___
*[`pkg/sql/pgwire/pre_serve.go` line 66 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299281022:-NcM22kgAhvOmW2MZp5A:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L66)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

```suggestion
		Help:        "Memory usage for SQL connections prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.
___
*[`pkg/sql/pgwire/pre_serve.go` line 72 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299281028:-NcM23-hDZXedSWwN5j2:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L72)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

```suggestion
		Help:        "Current memory usage for SQL connections prior to routing the connection to the target SQL server",
```
</blockquote></details>

Done.

**Review by knz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/109076)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stevendanna and @yuzefovich)

___
*[`pkg/sql/pgwire/pre_serve.go` line 72 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/109076#gh-1299281028:-NcMiktFDjXpzIOVXb70:bwmb33q) ([raw file](https://github.com/cockroachdb/cockroach/blob/045ec1562217c751684fdd4d907ee1402d2fb54b/pkg/sql/pgwire/pre_serve.go#L72)):*
<details><summary><i>Previously, stevendanna (Steven Danna) wrote…</i></summary><blockquote>

Looks like this one slipped through still.
</blockquote></details>

Indeed. Fixed.

**Review by stevendanna — APPROVED** [review]
Thanks!

**Comment by knz** [comment]
TFYR!

bors r=stevendanna

**Pull request merged**